src/coreComponents/math/UOM.hpp

```diff
@@ -0,0 +1,49 @@
+/*
+ * ------------------------------------------------------------------------------------------------------------
+ * SPDX-License-Identifier: LGPL-2.1-only
+ *
+ * Copyright (c) 2018-2020 Lawrence Livermore National Security LLC
+ * Copyright (c) 2018-2020 The Board of Trustees of the Leland Stanford Junior University
+ * Copyright (c) 2018-2020 TotalEnergies
+ * Copyright (c) 2019-     GEOSX Contributors
+ * All rights reserved
+ *
+ * See top level LICENSE, COPYRIGHT, CONTRIBUTORS, NOTICE, and ACKNOWLEDGEMENTS files for details.
+ * ------------------------------------------------------------------------------------------------------------
+ */
+
+/**
+ * @file UOM.hpp
+ * @brief Regroups useful constants and functions relative to Units Of Measures
+ * that are globally used for math and physics computations.
+ */
+#ifndef GEOS_MATH_UOM_HPP_
+#define GEOS_MATH_UOM_HPP_
+
+namespace geos
+{
+
+namespace constants
+{
+
+/// @brief Zero degree Celsius in Kelvin
+constexpr double zeroCInK = 273.15;
```

Comment:
```suggestion
constexpr double zeroDegreesCelsiusInKelvin = 273.15;
```
to stick to our convention of spelling things out entirely